haystack/nodes/retriever/web.py

```diff
@@ -51,6 +51,7 @@ def __init__(
         api_key: str,
         search_engine_provider: Union[str, SearchEngine] = "SerperDev",
         allowed_domains: Optional[List[str]] = None,
+        link_content_fetcher: Optional[LinkContentFetcher] = None,
```

Comment:
I would put this after all the other the other existing arguments, if someone is initialising it like this it could fail:
```
WebRetriever("api_key", "SerperDev", [], 20)
```

Reply:
I totally get your point but we already screwed up the order with `allowed_domains`, wdyt we should do now? :-)

Reply:
We should make a PR to fix that too. It's still not been released so we're fine.

Reply:
Ok, deal. Let's integrate this one now and then make a PR to fix the order, cool @silvanocerza ?

Reply:
Deal. Approving. 👍